styleguide/components/61-layouts/general/info-layout/_detail-layout.scss

```diff
@@ -0,0 +1,34 @@
+.detail-layout {
+  @extend %content-container;
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin-top: 1.2rem;
+  font-size: .9rem;
```

Comment:
Isn't there's a mixin/class/thingy to handle text sizes?
I feel this should be handled on a higher level.

Reply:
I'll create a refactor ticket for this. It doesn't belong in this branch.

Reply:
I've created ticket DTGB-435 for this purpose.